_source/user-guide/infrastructure-monitoring/explore-in-kibana.md

```diff
@@ -12,14 +12,12 @@ contributors:
   - daniel-tk
 ---
 
-Drilldown links are shortcuts that take you directly to the relevant Kibana results in your logs.
-When you identify an issue in your graph and want to investigate it further,
-you can click a drilldown link
-and go straight to the related logs.
+**Explore in Kibana** are drilldown links that give you a direct shortcut from your Metrics' visualizations to the relevant logs in Kibana.
+
+You'll find that many of the dashboards provided out-of-the-box by Logz.io come with **Explore in Kibana** drilldown links preconfigured. But you can also add your own to any dashboards and visualizations. (The only caveat is you'll need to have edit-permissions to the dashboard, so if you don't, duplicate the dashboard first.)
```

Comment:
That's not true unfortunately.
We don't have pre-configured explore in kibana links, as they are pretty specific per user. It is mostly something that they need to do own their own (that's why in the guide I wrote there are detailed examples and explanations - so they will have a reference)